jenkins/swift-functional-tests/10-install-ring.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -xue
+
+
+function amend_apache_conf {
+    local option=$1
+    local filepath
+    for filepath in "/etc/apache2/apache2.conf" "/etc/httpd/conf/httpd.conf"; do
+        if  [[ -f $filepath ]]; then
+            sudo sed -i'.sedbck' "s/KeepAlive O.*/KeepAlive ${option}/" $filepath
```

Comment:
No especially because this script is for "test and dev purpose only". But I am not the original author of this piece of beauty. A change here should then be forward ported to master. It's not worth it, imo.